internal/worker/server.go

```diff
@@ -28,6 +28,15 @@ import (
 	"github.com/osbuild/osbuild-composer/internal/worker/clienterrors"
 )
 
+const (
+	WorkerJobTypeOSBuild        string = "osbuild"
+	WorkerJobTypeOSBuildKoji    string = "osbuild-koji"
```

Comment:
Yes, we could. I didn't define a new type, because it didn't seem to add too much value. The usual downside of doing so is that you ten can not use the new type easily with functions which take `string` as an argument. However I'm currently not aware of us doing this with the JobType anywhere so it should be OK to do...

Reply:
The idea crossed my mind too, but probably doesn't add much value.

Reply:
So after looking into it more, I decided to not do it. We do call `strings.HasPrefix()` on the job type, which would require casting the value to `string`. Also Prometheus metrics rely on the value being `string`. While it's all doable, I feel like it would just complicate things without bringing too much value.

Reply:
It would help IDEs even more. But yeah, let's not make this PR more complex, we can revisit this idea in the future if needed.